charts/seed-terraformer/charts/packet-infra/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Delete this file (the whole `packet-infra` directory).

Reply:
Done